src/Models/CacheKey.php

```diff
@@ -99,7 +99,7 @@ public static function remove(DataObject $dataObject): void
         }
     }
 
-    public static function generateKeyHash(DataObject $dataObject): string
+    protected static function generateKeyHash(DataObject $dataObject): string
```

Comment:
Only internal usage. Went with `protected` as I figured there's a small chance devs might extend this Model.